README.md

```diff
@@ -33,6 +33,11 @@ Mixins can be imported directly
 import { SvgIcon } from '@hypothesis/frontend-shared';
 ```
 
+
+### Related documents
+
+[Development](https://github.com/hypothesis/frontend-shared/blob/main/docs/developing.md)
+
 ### License
 
 The Hypothesis client is released under the [2-Clause BSD License][bsd2c],
```

Comment:
Unrelated to the PR, but the text "The Hypothesis client is released under" is incorrect for this repository. The license is documented in `package.json` metadata so I'd suggest you can just remove this section. Please keep that for a separate PR though.